caikit/runtime/http_server/__init__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright The Caikit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Local
+from .http_server import HEALTH_ENDPOINT, RuntimeHTTPServer
+from .pydantic_wrapper import dataobject_to_pydantic, pydantic_to_dataobject
```

Comment:
I think we also want to add a `main` clause to this module. I'm not _entirely_ sure the right way to do that, but we want to make sure that `python -m caikit.runtime.http_server` still works. I think it would either be:

* Add a `__main__.py` file that invokes `http_server.http_server.main`
* Add an `if __name__ == "__main__":` clause in this file.

Reply:
Also, this hasn't changed in this PR, but I think we need to update the import logic in `caikit/runtime/__main__.py` as well. Currently it [proactively imports the HTTP server](https://github.com/caikit/caikit/blob/main/caikit/runtime/__main__.py#L10C40-L10C57) which would mean that you can't use that entrypoint to run just the `grpc_server` if you didn't have the `runtime-http` extras set installed.

Reply:
I added a `__main__.py` file for the first point. 

For the second point, I am planning to open a separate issue and have a subsequent PR!

Reply:
Beautiful!